src/main/java/lingogo/model/flashcard/FlashcardInGivenFlashcardListPredicate.java

```diff
@@ -23,6 +27,19 @@ public FlashcardInGivenFlashcardListPredicate(List<Flashcard> givenFlashcardList
         this.givenFlashcardSet = tempSet;
     }
 
+    /**
+     * Constructs a predicate that will check if a specified flashcard
+     * is in the list of specified flashcards.
+     * @param flashcards The list of specified flashcards.
+     */
+    public FlashcardInGivenFlashcardListPredicate(Flashcard... flashcards) {
+        HashSet<Flashcard> tempSet = new HashSet<>();
+        for (Flashcard flashcard : flashcards) {
+            tempSet.add(flashcard);
+        }
+        this.givenFlashcardSet = tempSet;
+    }
```

Comment:
We can just use the constructor above? (just create a `new List(phrase1, phrase2, ...)` inside the test)